osu.Game/Rulesets/RulesetStore.cs

```diff
@@ -120,6 +135,21 @@ private void loadFromAppDomain()
             }
         }
 
+        private void loadUserRulesets()
+        {
+            try
```

Comment:
Probably should be done over each individual ruleset instead, and the error should contain the ruleset's name. No reason to fail loading all other rulesets if one's broken.

Reply:
This may not be needed actually, as the exception is catched internally in `loadRulesetFromFile()`